frontend/src/index.jade

```diff
@@ -132,13 +141,15 @@ html
                     right: (((user.commits.length-j-1)/user.commits.length) * 100) + '%'\
                   }"
                 )
-              .summary-chart__contrib(
-                v-bind:title=
-                    "'Total contribution from ' + minDate + ' to ' + maxDate + ': ' + user.totalCommits + ' lines'"
-               )
+              .summary-chart__contribution(
+                  v-for="(widths, fileType) in getContributionBars(user.fileTypeContribution)"
+              )
                 .summary-chart__contrib--bar(
-                  v-for="width in getContributionBars(user.totalCommits)",
-                  v-bind:style="{ width: width+'%' }"
+                  v-for="width in widths",
+                  v-bind:style="{ width: width + '%', 'background-color'\
+                      : contributionBarColors[user.repoPath][fileType]}",
```

Comment:
perhaps use a higher level break, at the comma, joining the `background-color` to the next line?